.gitignore

```diff
@@ -1,29 +1,6 @@
-# temporary files
-*~
-
-# Compiled Object files
-*.slo
-*.lo
-*.o
-
 # Python compiled files
 *.pyc
 
```

Comment:
with Python 3 we have `__pycache__` directories instead. Maybe we can remove that one, too.